src/transformers/integrations.py

```diff
@@ -676,24 +697,56 @@ def setup(self, args, state, model, **kwargs):
         variables:
 
         Environment:
-        - **WANDB_LOG_MODEL** (`str`, *optional*, defaults to `"false"`):
-            Whether to log model and checkpoints during training. Can be `"end"`, `"checkpoint"` or `"false"`. If set
-            to `"end"`, the model will be uploaded at the end of training. If set to `"checkpoint"`, the checkpoint
-            will be uploaded every `args.save_steps` . If set to `"false"`, the model will not be uploaded. Use along
-            with [`~transformers.TrainingArguments.load_best_model_at_end`] to upload best model.
-
-            <Deprecated version="5.0">
-
-            Setting `WANDB_LOG_MODEL` as `bool` will be deprecated in version 5 of 🤗 Transformers.
-
-            </Deprecated>
-        - **WANDB_WATCH** (`str`, *optional* defaults to `"false"`):
-            Can be `"gradients"`, `"all"`, `"parameters"`, or `"false"`. Set to `"all"` to log gradients and
-            parameters.
-        - **WANDB_PROJECT** (`str`, *optional*, defaults to `"huggingface"`):
-            Set this to a custom string to store results in a different project.
+        - **WANDB_ANONYMOUS** (`str`, *optional*, defaults to `None`):
```

Comment:
```suggestion
        - **WANDB_ANONYMOUS** (`str`, *optional*):
```
We never write the default when it's `None` (as it is strongly implied by optional).